core/src/main/scala/kafka/common/TopicAndPartition.scala

```diff
@@ -37,5 +35,9 @@ case class TopicAndPartition(topic: String, partition: Int) {
 
   override def toString = "[%s,%d]".format(topic, partition)
 
-  def toJson = Json.encode(Map("version" -> version, "topic" -> topic, "partition" -> partition))
+  def toJson = Json.encode(Map("version" -> TopicAndPartition.version, "topic" -> topic, "partition" -> partition))
+}
+
+object TopicAndPartition {
+  val version: Long = 1L
```

Comment:
I'm wondering can we remove this version? IsrChangeNotificationListener does not use it. And It seems not used anywhere else either... I kept it simply because it was there before.